### ISSUE TITLE
[Feature request] Long-press FAB to refresh timeline and go to top

### ISSUE THREAD

**Issue by AkazaRenn** [tuskyapp/Tusky#1859]
That could be useful since the top tabs are not very reachable. 

* * * *
- Tusky Version: 11.0
- Android Version: 9
- Android Device: Xperia Ace

- [ x ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by charlag** [comment]
Is this relevant since you can move tabs to the bottom? (so far in nightly)
I think it's really hard to discover.

**Comment by AkazaRenn** [comment]
@charlag I agree, the reason I'm used to such operation is that I use Share for Weibo and Flamingo for Twitter, both of which support it. I thought it was a common sense 😅

But still, even if I move the tabs to bottom I would need to do

1. Tap the tab to go to top
2. Pull the page to refresh
3. Tap the tab again to go to top

Would be nice to have something doing them all in one click.

**Comment by charlag** [comment]
@AkazaRenn I think mixing it with fab is not great but long press on the tab? Maybe
